tlspuffin/build.rs

```diff
@@ -1,15 +0,0 @@
-use std::process::Command;
-
-fn main() {
-    if cfg!(feature = "asan") {
-        let output = Command::new("clang")
-            .args(["--print-resource-dir"])
-            .output()
-            .expect("failed to clang to get resource dir");
-        let clang: &str = std::str::from_utf8(&output.stdout).unwrap().trim();
-
-        println!("cargo:rustc-link-arg=-Wl,-rpath={}/lib/linux/", clang);
-        println!("cargo:rustc-link-arg=-fsanitize=address");
-        println!("cargo:rustc-link-arg=-shared-libasan");
-    }
-}
```

Comment:
This file has to stay the `cargo:rustc-link-lib=asan` instructions have to go away.

This also means that this branch is not needed: https://github.com/tlspuffin/openssl-src-rs/tree/fuzz-asan